applications/dotwise.md

```diff
@@ -0,0 +1,191 @@
+
+  **P.S:** This mockups designed for inspiration. The final design can be different from this mockups.
+
+- Data models / API specifications of the core functionality
+
+  **Core API:** Dotwise Core API will be developed with [Fast API](https://fastapi.tiangolo.com/) and it will be used for core functionalities. It will be used by the frontend and backend services.
+
+  **SubQuery API:** SubQuery API will be used for querying the data from the Substrate-based networks.
+
+- An overview of the technology stack to be used
+
+  ![techstack](https://github.com/tolgayayci/Grants-Program/assets/40897846/21ed9c58-1950-4e8a-bf28-9e484a1eaf4b)
+
+  **Subquery:** Subquery is a tool for building custom data indexes for Polkadot, Kusama, and other Substrate-based networks. It is a great tool for building custom data indexes. It provides a great developer experience with many built-in features such as GraphQL support, TypeScript support, and Substrate support. It also has a great community and ecosystem with many plugins and tools.
+
+  **Fast API:** FastAPI is a modern, fast (high-performance), web framework for building APIs with Python 3.7+ based on standard Python type hints. That is fully compatible with OpenAPI 3.0 (formerly known as Swagger). It is a great tool for building APIs with Python.
+
+- Documentation of core components, protocols, architecture, etc. to be deployed
+  - General Guides & Documentation
+  - Test Coverage Reports
+
+- What your project is _not_ or will _not_ provide or implement
+  - There isn't any support for parachains that SubQuery doesn't support
+  - SQL based queries not supported, only GraphQL
+
+### Ecosystem Fit
+
+- Where and how does your project fit into the ecosystem?
+
+  Polkadot ecosystem doesn't have a solution like Dune Analytics and this projects aims start to fill this gap. Users can explore and analyze the data on the Polkadot & Kusama networks with custom queries and visualizations easily and share their queries or dashboards with the community.
+
+- Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+
+  Who needs to query and visualize Polkadot on-chain data easily is target base of Dotwise.
+
+- What need(s) does your project meet?
+
+  - Users should be able to query and visualize the data on the Polkadot & Kusama networks easily. 
+  - Users can create graphql queries using the built-in query builder.
+  - Users who are not familiar with GraphQL can use the built-in query builder to create queries just use the select boxes and input fields easily.
+
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  
+  No, there isn't any similar project in the Polkadot ecosystem but there is developing solutions about data analysis as my knowledge. So if there will be any good data source ı can rich the my data sources and make the platform more useful.
```

Comment:
I hadn't heard of the Web3Go project, I looked it up when you mentioned it, and it was first applied about two years ago, so I may have overlooked it. Although it is similar in proposal, as I have stated in my mock ups, when my project ends, it will have a working logic similar to Dune Analytics and I can say that it will be used in a simpler way.

Another difference is that Dotwise uses graphql and non-technical users can create queries by simply selecting attributes. Additionally, except for query and visualization, widgets like text, image can be created in the form of drag&drop on a dashboard.